custom-connectors/EleadProductReference/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#1F1A3C",
+    "capabilities": [],
+    "publisher": "Bhargava, Aman Avinash"
```

Comment:
Please also add the `stackOwner` property, and remove any PII. The publisher and stackowner fields should point to the company, not an individual, for certified connnectors.